### PR TITLE
Make info about kde correct

### PR DESCRIPTION
**Pull request by CarlSchwan**
And add inline reply as optional feature that de implements

### CODE REVIEW

File: _desktop-environments/gnome [hunk 1]
```diff
@@ -20,6 +20,7 @@ notifications:
   notification-center: true
   images: true
   replace: true
+  inline-reply: true
```
Comment (CarlSchwan) [inline _desktop-environments/gnome:23]:
```suggestion
  inline-reply: false
```

File: _desktop-environments/mate [hunk 2]
```diff
@@ -19,6 +19,7 @@ notifications:
   notification-center: false
   images: true
   replace: false
+  inline-reply: true
```
Comment (CarlSchwan) [inline _desktop-environments/mate:22]:
```suggestion
  inline-reply: false
```

File: _desktop-environments/pantheon [hunk 3]
```diff
@@ -20,6 +20,7 @@ notifications:
   notification-center: true
   images: true
   replace: true
+  inline-reply: true
```
Comment (CarlSchwan) [inline _desktop-environments/pantheon:23]:
```suggestion
  inline-reply: false
```

File: _desktop-environments/ubuntu [hunk 4]
```diff
@@ -20,6 +20,7 @@ notifications:
   notification-center: true
   images: true
   replace: true
+  inline-reply: true
```
Comment (CarlSchwan) [inline _desktop-environments/ubuntu:23]:
```suggestion
  inline-reply: false
```

File: _desktop-environments/xfce [hunk 5]
```diff
@@ -19,6 +19,7 @@ notifications:
   notification-center: true
   images: true
   replace: true
+  inline-reply: true
```
Comment (CarlSchwan) [inline _desktop-environments/xfce:22]:
```suggestion
  inline-reply: false
```

**Comment by cassidyjames** [comment]
Thanks, @CarlSchwan! Can you explain how Inline Reply is a replacement for a System Tray icon? Do some apps implement inline replies in their system tray menus?

**Comment by cassidyjames** [comment]
@CarlSchwan great, thank you!

**Review by cassidyjames — APPROVED** [review]

**Pull request merged**